doc/conditional_independence.rst

```diff
@@ -0,0 +1,180 @@
+.. _conditional_independence:
+
+========================
+Conditional Independence
+========================
+
+.. currentmodule:: dodiscover.ci
+
+Testing for invariances in the data, such as conditional independence (CI) can be represented graphically
+in the form of d-separation statements under the causal faithfulness assumption. Given this, one
+is interested in high-powered and well-controlled CI tests that can be used to test for CI in data.
+
+CI tests are framed as a statistical hypothesis test, with the following null hypothesis for a given
+pair of variables ``(X, Y)`` and a conditioning set ``Z`` (which may be empty):
+
+:math:`H_0: X \perp Y | Z`, or written in terms of their distribution :math:`H_0: P(Y | X, Z) = P(Y | Z)`
+
+Similarly, the alternative hypothesis is written as:
+
+:math:`H_0: X \not\perp Y | Z`, or written in terms of their distribution :math:`H_0: P(Y | X, Z) \neq P(Y | Z)`
```

Comment:
Shouldn't this be H_1 then? One should also emphasize that it is impossible to reject this in practice and, thus, we can only reject independence.

Reply:
Yes good catch fixed.